vagrant/injector-cluster.hosts

```diff
@@ -0,0 +1,6 @@
+[hyperfoil_controller]
```

Comment:
we want this info committed?

Reply:
Yes the groupname is necessary.

Reply:
Add all your vagrant specific files with info like IP's  (`192.168.42.45` and so on) to gitignore and remove from the PR

Reply:
This file has been removed.